spec/requests/conversion_hosts_spec.rb

```diff
@@ -45,4 +45,198 @@
       expect(response).to have_http_status(:forbidden)
     end
   end
+
+  context "create" do
+    let(:vm) { FactoryGirl.create(:vm) }
+    let(:host) { FactoryGirl.create(:host) }
+
+    let(:sample_conversion_host_from_vm) do
+      {
+        :name          => "test_conversion_host_from_vm",
+        :resource_type => "vm",
```

Comment:
shouldn't `:resource_type` be `vm.class.name`? so that you don't care what class/type the factory assigned to the `:vm`?